### ISSUE TITLE
Cannot create HTTP profiles in v15.0

### ISSUE THREAD

**Issue by sporky** [F5Networks/f5-appsvcs-extension#149]
#### Do you already have an issue opened with F5 support?
No

##### ISSUE TYPE
 - Bug Report

##### AS3 BUILD/ VERSION
```
3.14.0-4
```

##### BIGIP VERSION
```
Sys::Version
Main Package
  Product     BIG-IP
  Version     15.0.1
  Build       0.0.11
  Edition     Final
  Date        Fri Aug  2 21:38:03 PDT 2019
```

##### SUMMARY

```
Declarations that include HTTP profiles aren't created on the BIGIP as of v15.  Looks like the request/response chunking options in the bigip.conf have changed..  Preserve and Selective are out and have been replaced by "sustain".
```

##### STEPS TO REPRODUCE

Stick the following declaration in a configmap and save to magic.yaml
kubectl apply -f magic.yaml

```
    {
      "class": "AS3",
      "action": "deploy",
      "persist": true,
      "declaration": {
        "class": "ADC",
        "schemaVersion": "3.11.0",
        "id": "APP-000001",
        "label": "stage_test_vs_com_ipv4_as3test",
        "remark": "AS3 cis_https_waf ver 1.0",
        "test": {
          "class": "Tenant",
          "test_app": {
            "class": "Application",
            "template": "generic",
            "test_as3test": {
              "class": "Service_HTTPS",
              "virtualAddresses": [
                "172.10.10.10"
              ],
              "virtualPort": 443,
              "profileHTTP": {
                "use": "stage_test_vs_com_ipv4_as3test_profile_http"
              },
              "redirect80": false,
              "snat": "auto",
              "pool": "stage_test_vs_com_ipv4_as3test_pool",
              "serverTLS": {
                "bigip": "/Common/clientssl"
              },
              "clientTLS": {
                "bigip": "/Common/serverssl"
              }
            },
            "stage_test_vs_com_ipv4_as3test_pool": {
              "class": "Pool",
              "members": [
                {
                  "servicePort": 443,
                  "serverAddresses": [],
                  "shareNodes": true
                }
              ]
            },
            "stage_test_vs_com_ipv4_as3test_profile_http": {
              "class": "HTTP_Profile",
              "xForwardedFor": true
            }
          }
        }
      }
    }
```

##### EXPECTED RESULTS

```
Expected to see a new tenant, app, virtual server and http profile.
```

##### ACTUAL RESULTS

Confusion, mostly.  No new config was built.

```
From restnoded.log
severe: [appsvcs] {"message":"Declaration failed: 01070281:3: Invalid \\\"http response chunking\\\" value for profile /test/test_app/stage_test_vs_com_ipv4_as3test_profile_http. Invalid entry, check allowed values.","level":"error"}

```

**Comment by dstokesf5** [comment]
I have confirmed this bug exists on 15.0 (but not previous versions), and I have added it to our internal product backlog as AT-750.

**Comment by ianwijaya** [comment]
when will this be resolved? I can't proceed with 15.0 because of this issue.

**Comment by dstokesf5** [comment]
3.16.0 has been released that fixes this issue.
